src/Shared/ResultsHelpers/FileResultHelper.cs

```diff
@@ -52,6 +53,40 @@ internal static async Task WriteFileAsync(HttpContext context, Stream fileStream
         }
     }
 
+    internal static async Task WriteFileAsync(HttpContext context, ReadOnlyMemory<byte> buffer, RangeItemHeaderValue? range, long rangeLength)
+    {
+        var outputStream = context.Response.Body;
+
+        try
+        {
+            if (range == null)
```

Comment:
```suggestion
            if (range is null)
```

It's on our todo list to enforce: https://github.com/dotnet/aspnetcore/issues/39323